Shared/sdk/net/SyncStructures.h

```diff
@@ -2065,6 +2069,7 @@ struct SWorldSpecialPropertiesStateSync : public ISyncStructure
         if (bitStream.Can(eBitStreamVersion::WorldSpecialProperty_FireballDestruct))
             bitStream.WriteBits(reinterpret_cast<const char*>(&data2), BITCOUNT2);
 
+        if (bitStream.Can(eBitStreamVersion::WorldSpecialProperty_ExtendedWaterCannons))
         if (bitStream.Can(eBitStreamVersion::WorldSpecialProperty_RoadSignsText))
             bitStream.WriteBits(reinterpret_cast<const char*>(&data3), BITCOUNT3);
```

Comment:
Merge error? Looks like you're missing a WriteBits call for the first if-statement.

Reply:
Some strange things happened during conflict resolution. I've already fixed everything